sql/core/src/test/scala/org/apache/spark/sql/execution/debug/DebuggingSuite.scala

```diff
@@ -69,16 +69,17 @@ class DebuggingSuite extends SharedSparkSession with DisableAdaptiveExecutionSui
     }
 
     val output = captured.toString()
+    val hashedModeString = "HashedRelationBroadcastMode(List(input[0, bigint, false]),false)"
     assert(output.replaceAll("\\[id=#\\d+\\]", "[id=#x]").contains(
-      """== BroadcastExchange HashedRelationBroadcastMode(List(input[0, bigint, false])), [id=#x] ==
-        |Tuples output: 0
-        | id LongType: {}
-        |== WholeStageCodegen (1) ==
-        |Tuples output: 10
-        | id LongType: {java.lang.Long}
-        |== Range (0, 10, step=1, splits=2) ==
-        |Tuples output: 0
-        | id LongType: {}""".stripMargin))
+      s"""== BroadcastExchange $hashedModeString, [id=#x] ==
+         |Tuples output: 0
```

Comment:
can we revert these unnecessary indentation changes?

Reply:
is it ok with code style